readme.md

```diff
@@ -44,6 +44,12 @@ Returns an array of objects in the format `{ pattern: string, opts: Object }`, w
 
 Note that you should avoid running the same tasks multiple times as they contain a file system cache. Instead, run this method each time to ensure file system changes are taken into consideration.
 
+### globby.hasMagic(pattern, [options]);
```

Comment:
Please note my https://github.com/sindresorhus/globby/issues/35#issuecomment-256328382, maybe we should modify the signature–sorry for the clutter.